src/reynir_correct/main.py

```diff
@@ -258,6 +261,8 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
         for t in tokenize(gen(args.infile), **options):
             # Normal shallow parse, one line per sentence,
             # tokens separated by spaces
+            # Note this uses the tokenize function in errtokenizer.py
+            # instead of the one in Tokenizer
```

Comment:
We could potentially make this explicit by calling errtokenizer.tokenize() - if there is a danger of confusion (which there might well be).

Reply:
Changed import name to errtokenize